src/libponyrt/sched/scheduler.c

```diff
@@ -1076,7 +1076,7 @@ pony_ctx_t* ponyint_sched_init(uint32_t threads, bool noyield, bool nopin,
 
   // If minimum thread count is > thread count, cap it at thread count
   if(min_threads > threads)
-    min_threads = threads;
+    min_threads = threads; // this becomes the equivalent of --ponynoscale
```

Comment:
Should we remove this entirely now then?

Reply:
As, with the current implementation, this would be triggered in such a case:
```
$ ./helloworld --ponyminthreads=2048                                                
Hello, world.
```

Would it be better to bail out here instead? wdyt?

Reply:
Here's a proposal: an explicit check against physical cores:

```
$ ./helloworld --ponyminthreads=2048
Can't have --ponyminthreads > physical cores (2048 > 6)
```

I _think_ this makes the most sense.